src/mbgl/renderer/sources/render_image_source.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <mbgl/renderer/buckets/raster_bucket.hpp>
```

Comment:
Forward declare `RasterBucket` (and define `~RenderImageSource` in the .cpp file) instead.